Creating OCP performance guide (#564)
RedHatInsights/red-hat-ansible-automation-platform-documentation: downstream/modules/platform/ref-container-resource-requirements.adoc

[id="ref-container-resource-requirements"]

= Containers resource requirements

> You can configure the resource requirements of task and the web containers, at both the lower end (requests) and the upper end (limits). The execution environment control plane is used for project updates, but is normally the same as the default execution environment for jobs.

Reviewer: ```suggestion
You can configure the resource requirements of tasks and the web containers, at both the lower end (requests) and the upper end (limits). The execution environment control plane is used for project updates, but is normally the same as the default execution environment for jobs.
```

Not sure if this suggestion is necessary

> Setting resource requests and limits is a best practice because when a container has both defined, it is given a higher _Quality of Service_ class. 

Reviewer: ```suggestion
Setting resource requests and limits is a best practice because a container that has both defined is given a higher _Quality of Service_ class. 
```
